pytest.ini

```diff
@@ -4,3 +4,13 @@ log_format = %(asctime)s - %(levelname)s - %(name)s - %(message)s
 log_level = DEBUG
 markers =
     java
+filterwarnings =
+# individual warnings filters are specified as a sequence of fields separated by colons:
+# action:message:category:module:line
+# <reference: https://docs.python.org/3/library/warnings.html#warning-filter>
+    # ignore third-party modules's warns
+    ignore:encoding is deprecated, Use raw=False instead.:PendingDeprecationWarning:.*msgpack_numpy
+    # ignore warnings for users
+    ignore::DeprecationWarning:allennlp\.common\.registrable
```

Comment:
I don't think we should do this.  I think we should instead fix the warnings, so that our own library isn't using deprecated code.